### PR TITLE
Docker build procedure added 

### PR DESCRIPTION
**Pull request by orion-belt**
* Added docker build procedure
* Added sample docker-compose file

### CODE REVIEW

**Comment by bhack** [comment]
Can we coordinate with https://github.com/srsran/srsRAN_Project/pull/14 ?

**Comment by asaezper** [comment]
Thanks for this! We are already using something similar for testing and developing. We are planning to release it soon.

File: docker/entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    # render template and inline replace config file
+    sed -i "${EXPRESSIONS}" ${c}
+done
+
+echo -e  "Done setting the configuration\n"
+
+echo -e  "Running gNB Service \n"
+/srsran/bin/gnb --gnb_id $GNB_ID -c /srsran/etc/gnb_config.yaml
+
+# if [[ ${RUN_ZMQ_UE} == "yes" ]];then
```
Comment (bhack) [inline docker/entrypoint.sh:38]: What about the commented UE section? is it still only in the 4g repo?

Comment (asaezper) [reply docker/entrypoint.sh:38]: Yes, srsue will remain in the 4G repo

**Comment by asaezper** [comment]
Hey! We've already pushed a docker-compose to orchestrate a container with the gnb and also a container with open5gs. 
Hope it will be helpful for you! Please come back with any suggestions you may have.

**Pull request closed without merging**